sdk/aqueduct/tests/utils.py

```diff
@@ -149,8 +147,8 @@ def default_table_artifact(
         inputs=[],
         outputs=[artifact_id],
     )
-    dag = _construct_dag(
+    dag_module.__GLOBAL_DAG__ = _construct_dag(
```

Comment:
hmm because `_construct_operator()` is used by multiple test cases, I wonder if its safer to just use the local dag for the artifacts. These unit tests are not supposed to use decorators or the aqueduct client methods anyways, since its supposed to validate lower-level constraints.